### DISCUSSION TITLE
[flutterfire_ui] custom email validator

### DISCUSSION THREAD

**Discussion by komangnopha** [firebase/flutterfire#7638] [Feature request]
Hi, i am facing issue using flutterfire_ui, when i try to sign in using email that contains undescore or dash character, validator says "please provide valid email". Please help to resolve this issue. Thank You.

**Comment by lesnitsky** [comment]
@komangnopha this behavior will be changed after #7772 gets merged, so the underscore and dash character will be allowed.
